developer-guide/07-Plugins/03-plugin-as-python-package.md

```diff
@@ -340,6 +340,15 @@ include <PLUGIN_NAME>/LEADER_ONLY
 
 For an example, see the [logs2slack](https://github.com/Pioreactor/pioreactor-logs2slack) plugin's repository.
 
+## Install package from source
+
+It isn't enough to just `pip install` your package, as this will only add the script to the pioreactor system, not any extra configs you may have added. 
+If you want to install the whole package from source you can simply run:
+
+```
+pio plugins install <PLUGIN_NAME> --source <PLUGIN_FOLDER>
```

Comment:
PLUGIN_FOLDER should be the wheel file, no? Does it work when you point it at the folder (I think no)?

Reply:
It works when you point it at the folder! That's why I added these instructions. You don't need to build the wheel